File: src/test/java/io/binac/leetcode/ClimbingStairsTest.java
package io.binac.leetcode;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class ClimbingStairsTest {
    private final ClimbingStairs.Solution1 solution1 = new ClimbingStairs.Solution1();

    @Test
    void test1() {
        assertEquals(2, solution1.climbStairs(2));
        assertEquals(3, solution1.climbStairs(3));
    }
}

File: cpp/unittests/ClimbingStairsTest.cpp
#include "ClimbingStairs.h"
#include "gtest/gtest.h"

using namespace lcpp;

TEST(ClimbingStairs, Solution70_1) {
  EXPECT_EQ(2, Solution70_1().climbStairs(2));
  EXPECT_EQ(3, Solution70_1().climbStairs(3));
}